### DISCUSSION TITLE
Reason for using Decimal

### DISCUSSION THREAD

**Discussion by xiaoyual666** [fabianboesiger/ftx#105] [Q&A] [answered]
Hi,

I'm reading the code in the repo and noticed that it uses Decimal to represent price, for example the bid and ask prices. Is there a reason for using Decimal other than f64 in this repo?

**Comment by fabianboesiger** [comment, marked as answer]
Floating point numbers should never be used to represent prices. See [this](https://stackoverflow.com/questions/3730019/why-not-use-double-or-float-to-represent-currency) StackOverflow discussion for a technical reason why.

**Reply by xiaoyual666** [reply]
Thanks!
